ui/apps/platform/src/Containers/AccessControl/PermissionSets/ResourceDescription.tsx

```diff
@@ -61,6 +61,8 @@ const resourceDescriptions: Record<ResourceName, string> = {
         'Read: View all vulnerability report configurations. Write: Add, modify or delete vulnerability report configurations.',
     WatchedImage:
         'Read: View undeployed watched images monitored. Write: Configure watched images.',
+    WorkflowAdministration:
+        'Read: View all resource collections. Write: Add, modify, or delete resource collections.',
```

Comment:
@md2119 Does this description sound accurate, at least for this initial scope? Or do you recommend I change this to something more general than just "collections"?

Reply:
Current description lgtm. It aligns with the APIs.